pkgs/flox/default.nix

```diff
@@ -35,6 +37,8 @@ in
 
     postBuild = ''
       wrapProgram $out/bin/flox \
+        ${lib.optionalString (SENTRY_DSN != null) "--set SENTRY_DSN \"${SENTRY_DSN}\" "} \
+        ${lib.optionalString (SENTRY_ENV != null) "--set SENTRY_ENV \"${SENTRY_ENV}\" "} \
```

Comment:
doesnt this produce
```
wrapProgram $out/bin/flox \
       \
       \
       --set PKGDB_BIN       "${flox-pkgdb}/bin/pkgdb" \
       ...
```

if both are `null`? Not sure if that syntax is valid, it definitely looks weird

Reply:
If it's valid - no worries changing it!

Reply:
Yes, that is right, but works. I'm fine with how it looks, but if you want to change it please go ahead. Just keep it one line per variable, since we might be adding and removing things as we move forward.